state/protocol/errors.go

```diff
@@ -12,8 +12,14 @@ var (
 	// queried from a snapshot within the first epoch after the root block.
 	ErrNoPreviousEpoch = fmt.Errorf("no previous epoch exists")
 
-	// ErrNextEpochNotSetup is a sentinal error returned when
+	// ErrNextEpochNotSetup is a sentinel error returned when the next epoch
+	// has not been set up yet.
 	ErrNextEpochNotSetup = fmt.Errorf("next epoch has not yet been set up")
+
+	// ErrEpochNotCommitted is a sentinel error returned when the epoch has
+	// not been committed and information is queried that is only accessible
```

Comment:
Maybe more indicative to add `information is queried that is only accessible ....` to the error message as well.